src/RenderWebGL.js

```diff
@@ -1556,6 +1620,43 @@ class RenderWebGL extends EventEmitter {
         // Simplify boundary points using convex hull.
         return hull(boundaryPoints, Infinity);
     }
+
+    /**
+     * Sample a "final" color from an array of drawables at a given scratch space.
+     * Will blend any alpha values with the drawables "below" it.
+     * @param {twgl.v3} vec Scratch Vector Space to sample
+     * @param {Array<Drawables>} drawables A list of drawables with the "top most"
+     *              drawable at index 0
+     * @param {Uint8ClampedArray} dst The color3b space to store the answer in.
+     * @return {Uint8ClampedArray} The dst vector with everything blended down.
+     */
+    static sampleColor3b (vec, drawables, dst) {
```

Comment:
It looks like this function is relying on the caller to filter out hidden drawables, if desired. Is that accurate / intended?

Reply:
Yes and no - this is called inside of a tight loop, so I got rid of the visibility checks. 
The redundant checks are just wasted compute this deep.  I should probably document that it will work fine with any value of ghost, including 100, but not hidden/visible -- if "hide" it would still render here